examples/training/classification/imagenet/basic_training.py

```diff
@@ -147,7 +147,7 @@
 
 
 AUGMENT_LAYERS = [
-    keras_cv.layers.RandomFlip(),
+    keras_cv.layers.RandomFlip(mode="horizontal"),
```

Comment:
Instead, can we change the default mode for RandomFlip? vertical flip doesn't seem right to me in the first place

Reply:
Yes, I'll update the default in a separate PR. I think it's nice to make it explicit here anyway.